server/README.md

```diff
@@ -0,0 +1,74 @@
+The files are only read, chages won't persist.
+
+Alternatively, the container can also be told to use the [Local-File Backend][2] instead, which stores AAS and Submodels as individual JSON files and allows for persistent changes (except supplementary files, i.e. files referenced by `File` submodel elements).
+See [below](#options) on how to configure this.
+
+## Building
+The container image can be built via:
+```
+$ docker buildx build -t basyx-python-sdk-http-server .
+```
+
+## Running
+
+### Storage
+The container needs to be provided with the directory `/storage` to store AAS and Submodel files: AASX, JSON, XML or JSON files of Local-File Backend.
+
+This directory can be mapped via the `-v` option from another image or a local directory.
+To map the directory `storage` inside the container, `-v ./storage:/storage` can be used.
+The directory `storage` will be created in the current working directory, if it doesn't already exist.
+
+### Port
+The HTTP server inside the container listens on port 80 by default.
+To expose it on the host on port 8080, use the option `-p 8080:80` when running it.
+
+### Options
+The container can be configured via environment variables:
+- `API_BASE_PATH` determines the base path under which all other API paths are made available.
+  Default: `/api/v3.0`
+- `STORAGE_TYPE` can be one of `LOCAL_FILE_READ_ONLY` or `LOCAL_FILE_BACKEND`:
+  - When set to `LOCAL_FILE_READ_ONLY` (the default), the server will read and serve AASX, JSON, XML files from the storage directory.
+    The files are not modified, all changes done via the API are only stored in memory.
+  - When instead set to `LOCAL_FILE`, the server makes use of the [LocalFileBackend][2], where AAS and Submodels are persistently stored as JSON files.
+    Supplementary files, i.e. files referenced by `File` submodel elements, are not stored in this case.
+- `STORAGE_PATH` sets the directory to read the files from *within the container*. If you bind your files to a directory different from the default `/storage`, you can use this variable to adjust the server accordingly.
+
+### Running Examples
+
+Putting it all together, the container can be started via the following command:
+```
+$ docker run -p 8080:80 -v ./storage:/storage basyx-python-sdk-http-server
+```
+
+Since Windows uses backslashes instead of forward slashes in paths, you'll have to adjust the path to the storage directory there:
+```
+> docker run -p 8080:80 -v .\storage:/storage basyx-python-sdk-http-server
+```
+
+Per default, the server will use the `LOCAL_FILE_READ_ONLY` storage type and serve the API under `/api/v3.0` and read files from `/storage`. If you want to change this, you can do so like this:
+```
+$ docker run -p 8080:80 -v ./storage2:/storage2 -e API_BASE_PATH=/api/v3.1 -e STORAGE_TYPE=LOCAL_FILE_BACKEND -e STORAGE_PATH=/storage2 basyx-python-sdk-http-server
+```
+## Acknowledgments
```

Comment:
We should add here a section: 

``` 
### Docker Compose File
```

And link to the exemplary `compose.yaml`.

Reply:
I do not quite understand. What exemplary compose.yaml do you mean? Do you mean  there should be a tutorial on how to get the server to run using our compose.yaml?

Reply:
If you look at the `README`s of big projects, such as [nginx](https://hub.docker.com/_/nginx/) or [Nextcloud](https://hub.docker.com/_/nextcloud) (and scroll down a little bit), they all have a section with an example `compose` configuration. 
I think since that's so common, we should do the same and provide at least the one example we already have in our repo, if not more than that with different configurations.